### DISCUSSION TITLE
TortoiseORM Support in 10.x

### DISCUSSION THREAD

**Discussion by enchance** [fastapi-users/fastapi-users#1023] [General]
I'll try to work on a driver for **TortoiseORM** in 10.x.  Overall, I'm saddened that support was dropped. Coming from **Django to FastAPI**, it was one of the reasons I was able to transition easily.

If anyone has already started on the project let me know. I might be able to assist.

**Comment by caniko** [comment]
You welcome: https://github.com/Tobi-De/fastapi-users-tortoise

It is a superior ORM IMHO with respect to code aesthetics. The issue is that industry veterans don't want to switch away from `SQLAlchemy`, which is older, but also has a stable release, and is feature-complete.

Pydantic 2.0 will change the game significantly, let's wait and see :)
